packages/node_modules/@ciscospark/react-component-activity-item/src/index.js

```diff
@@ -42,6 +44,7 @@ const propTypes = {
 
 const defaultProps = {
   isAdditional: false,
+  // eslint-disable-next-line react/default-props-match-prop-types
```

Comment:
You could delete the `sdkInstance` since it is a required prop type. No need to add the eslint override.